src/commands/vault/transfer/processor.ts

```diff
@@ -86,63 +86,67 @@ export async function runTransferTreasurer({
-        style: "DANGER",
-        label: "Reject",
-      })
-    )
+  // send DM to submitter
+  // send DM to list treasurer but not requester, requester default approve
+  if (dataTransferTreasurerReq?.is_decided_and_executed === false) {
+    dataTransferTreasurerReq?.treasurer.forEach((treasurer: any) => {
+      if (treasurer.user_discord_id === i.user.id) {
+        return
+      }
+      const actionRow = new MessageActionRow().addComponents(
+        new MessageButton({
+          customId: `treaTransfer-approved-${dataTransferTreasurerReq?.request.id}-${dataTransferTreasurerReq?.request.vault_id}-${treasurer.user_discord_id}-${amount}-${token}-${chain}-${i.channelId}-${userId}`,
+          style: "SUCCESS",
+          label: "Approve",
+        }),
+        new MessageButton({
+          customId: `treaTransfer-rejected-${dataTransferTreasurerReq?.request.id}-${dataTransferTreasurerReq?.request.vault_id}-${treasurer.user_discord_id}-${amount}-${token}-${chain}-${i.channelId}-${userId}`,
+          style: "DANGER",
+          label: "Reject",
+        })
+      )
 
-    const cacheKey = `treaTransfer-${dataTransferTreasurerReq?.request.id}-${dataTransferTreasurerReq?.request.vault_id}-${treasurer.user_discord_id}-${address}-${amount}-${token}-${chain}-${i.channelId}`
+      const cacheKey = `treaTransfer-${dataTransferTreasurerReq?.request.id}-${dataTransferTreasurerReq?.request.vault_id}-${treasurer.user_discord_id}-${address}-${amount}-${token}-${chain}-${i.channelId}`
 
-    i.guild?.members.fetch(treasurer.user_discord_id).then((treasurer) => {
-      wrapError(i, async () => {
-        const msg = await treasurer.send({
-          embeds: [
-            composeEmbedMessage(null, {
-              title: `${getEmoji("ANIMATED_BELL", true)} mochi notifications`,
-              description: `<@${
-                i.user.id
-              }> has submitted the approval request #${
-                dataTransferTreasurerReq?.request.id
-              }\n${getEmoji("SHARE")} Send ${getEmoji(
-                token.toUpperCase() as keyof typeof emojis
-              )} ${amount} ${token} from ${vaultName} to ${destination} ${msgField}`,
-              color: msgColors.BLUE,
-              thumbnail: getAnimatedEmojiURL(emojis.ANIMATED_OPEN_VAULT),
-            }),
-          ],
-          components: [actionRow],
-        })
+      i.guild?.members.fetch(treasurer.user_discord_id).then((treasurer) => {
+        wrapError(i, async () => {
+          const msg = await treasurer.send({
+            embeds: [
+              composeEmbedMessage(null, {
+                title: `${getEmoji("ANIMATED_BELL", true)} mochi notifications`,
+                description: `<@${
+                  i.user.id
+                }> has submitted the request in ${vaultName} vault \n${getEmoji(
+                  "SHARE"
+                )} Send ${getEmoji(
+                  token.toUpperCase() as keyof typeof emojis
```

Comment:
chỗ này nhớ import `TokenEmojiKey` nhé
```suggestion
                )} Send ${getEmojiToken(
                  token.toUpperCase() as TokenEmojiKey
```